docs/getting-started/cluster-setup.md

```diff
@@ -0,0 +1,206 @@
+# Setting Up a Cluster
+You can setup a cluster in [Kubernetes](http://kubernetes.io/docs/whatisk8s/) or Jenkins.
+
+## Setting Up a Cluster on AWS using Kubernetes
+
+### Prerequisites
+- [kubectl](http://kubernetes.io/docs/user-guide/prereqs/)
+- an [AWS](http://aws.amazon.com) account
+- [AWS CLI](https://aws.amazon.com/cli/)
+- [Screwdriver tables on DynamoDB](datastore-setup/)
+
+### Create your cluster
+A cluster consists of a master API server and a set of worker VMs called nodes.
+Follow instructions at [Getting Started with AWS](http://kubernetes.io/docs/getting-started-guides/aws/).
+
+
+### Setup Kubernetes secrets
+A [Secret](http://kubernetes.io/docs/user-guide/secrets/) is an object that contains a small amount of sensitive data such as a password, a token, or a key. You will need to setup secrets for Screwdriver for your cluster to work properly.
+
+First, you will need to gather some secrets:
+
+| Secret Key        | Description |
+| :------------- |:-------------|
+| DATASTORE_DYNAMODB_ID | AWS Access Key ID |
+| DATASTORE_DYNAMODB_SECRET | AWS Secret Access Key |
+| SECRET_OAUTH_CLIENT_ID | The client ID used for [OAuth](https://developer.github.com/v3/oauth/) with Github |
+| SECRET_OAUTH_CLIENT_SECRET | The client secret used for OAuth with github |
+| SECRET_JWT_PRIVATE_KEY | A private key uses for signing JWT tokens. Can be anything |
+| SECRET_PASSWORD | A password used for encrypting session, and OAuth data. **Needs to be minimum 32 characters** |
+
+Here are some directions for getting your secrets:
+
+#### Get your DynamoDB secrets
+To get your `accessKeyId` and `secretAccessKey`:
+
+1. Navigate to [IAM](https://console.aws.amazon.com/iam) in your AWS console
+2. Click on Users, and create a Screwdriver user with `<USERNAME>`. _We recommend installing using an account which has full access to the AWS APIs._
```

Comment:
That's not what I would recommend. Why would they need full AWS access?

Also, "a Screwdriver user with `<USERNAME`>" is a weird phrase.